sitestats/sitestats-api/src/config/org/sakaiproject/sitestats/config/toolEventsDef.xml

```diff
@@ -129,7 +129,7 @@
 
 	<!-- news -->
 	<tool 
-		toolId="sakai.news"
+		toolId="sakai.simple.rss"
```

Comment:
If the eventId's overlap, I think you can just add this as additionalToolIds="sakai.simple.rss" and then the old events would still be retained. This is done for a few other tools.

Reply:
Yeah, good idea, I'll do that thanks.

Reply:
I added sakai.news as additionalToolId, but I don't think it really does anything unless you're still running the tool under that id. Everything in the database is keyed on the event ids themselves. If you don't use "sakai.simple.rss" as the main tool id, it doesn't show up in the UI either. But it shouldn't hurt to include it.